### PR TITLE
ETFE-1819 Check your answers page added

### PR DESCRIPTION
**Pull request by grahampaulcook**
Contains some TODO's as this links into previous pages not yet built, fine to go in before those pages and we can ensure those other pages PR's update the CYA logic.

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -122,33 +124,34 @@ destinationOffice.checkYourAnswersLabel = Were the goods received in Great Brita
 destinationOffice.error.required = Select if the goods were received in Great Britain or Northern Ireland
 destinationOffice.change.hidden = DestinationOffice
 
-selectReason.title.0 = Why are you alerting the consignor to an issue?
-selectReason.title.1 = Why are you rejecting the movement?
-selectReason.heading.0 = Why are you alerting the consignor to an issue?
-selectReason.heading.1 = Why are you rejecting the movement?
+selectReason.title.alert = Why are you alerting the consignor to an issue?
```
Comment (machinesleet) [inline conf/messages.cy:127]: Just searching the project for some of these messages, and can't find a few of them. Have I missed something, or should they be removed?

File: app/viewmodels/checkAnswers/SelectReasonSummary.scala [hunk 2]
```diff
@@ -26,27 +27,32 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
...
+    request.userAnswers.get(SelectReasonPage).map {
       answers =>
 
         val value = ValueViewModel(
           HtmlContent(
             answers.map {
               answer => HtmlFormat.escape(messages(s"selectReason.$answer")).toString
             }
-            .mkString(",<br>")
+            .mkString("<br><br>")
```
Comment (machinesleet) [inline app/viewmodels/checkAnswers/SelectReasonSummary.scala]: As mentioned, I wonder whether this should be something like [this](https://github.com/hmrc/emcs-tfe-explain-shortage-excess-frontend/pull/48/files) to match the prototype?

File: conf/messages.cy (hunk 1 above)
Comment (grahampaulcook) [reply conf/messages.cy:127]: @machinesleet Pushed a commit to fix that, that one should be used but was an incorrect on the view https://github.com/hmrc/emcs-tfe-alert-rejection-frontend/pull/22/commits/6aa50a2d858ef3bf2ab5bb9674e6a7fcb92e6552

File: app/viewmodels/checkAnswers/SelectReasonSummary.scala (hunk 2 above)
Comment (grahampaulcook) [reply app/viewmodels/checkAnswers/SelectReasonSummary.scala]: fixed with commit https://github.com/hmrc/emcs-tfe-alert-rejection-frontend/pull/22/commits/fcb47366a5ef537a879e9b6800c787bd39292f10

**Review by machinesleet — APPROVED** [review]

**Pull request merged**